python-api/gstswitch/helpers.py

```diff
@@ -134,10 +137,6 @@ def get_test_video(self):
         """Returns a list of processes acting as video test sources running
         :returns: A list containing all video test sources running
         """
-        i = 0
-        for test in self._running_tests_video:
-            print(i, "pattern:", test.pattern)
-            i += 1
         return self._running_tests_video
```

Comment:
Now I think about it, this should probably be 
```python
return list(self._running_tests_video)
```
to create a copy of the list. That way the caller can't modify the internals of this class by accident.